std/experimental/allocator/building_blocks/bucketizer.d

```diff
@@ -166,11 +166,11 @@ struct Bucketizer(Allocator, size_t min, size_t max, size_t step)
     static if (hasMember!(Allocator, "owns"))
     Ternary owns(void[] b)
```

Comment:
Shouldn't this be `const`?

Reply:
Yes it should. Apparently the cherry pick didn't go as smooth as I had hoped 😄